growth/mailerAPI.js

```diff
@@ -0,0 +1,55 @@
+//API code from Growth Team: This code is to be implemented where a diiferent team would like to use
+//our API to send out emails
+// file name: mailerAPI.js
+function sendMail(emailAddress, emailSubject, emailHtmlPATH) {
+    const data = {
+        "mail": {
+            "to": [
+                emailAddress
+            ],
+            "cc": "",
+            "bcc": "",
+            "subject": emailSubject,
+            "html": emailHtmlPATH,
+        },
+        "isScheduled": "off",
+        "timeToSend": "",
+    }
+
+// API DOMAIN NEEDS TO BE CHANGED TO RENDER DOMAIN WHEN USED BY THE IAM GROUP
+    fetch('https://mail-service-69zm.onrender.com/api/mail/sendMail', {
```

Comment:
`https://mail-service-69zm.onrender.com/api/mail/sendMail` move to constants file